src/Sylius/Component/Core/Formatter/StringInflector.php

```diff
@@ -45,8 +45,4 @@ public static function nameToUppercaseCode($value)
     {
         return strtoupper(self::nameToCode($value));
     }
-
-    private function __construct()
```

Comment:
Hm, not sure about that if that can be fixed using a private constructor. Besides it doesn't make a lot sense to instantiate this class anyway :)